lib/http/utils/create-wallet.js

```diff
@@ -6,7 +6,7 @@ module.exports = function createEncryptedWallet(accountId, password) {
 
   return wallet.encrypt(password).then(res => {
     return new Promise((resolve, reject) => {
-      fs.writeFile(`../../keys/${accountId}.json`, res, { flag: 'ax' }, (err) => {
+      fs.writeFile(`./keys/${accountId}.json`, res, { flag: 'ax' }, (err) => {
```

Comment:
we need to make this use an absolute path otherwise it will break depending on how we run it. 
can you use something like 
```
path.join(__dirname, '../../../', keys, `${accountId}.json`)
``` 
(we actually should also escape the accountId - but we trust the client for now :)